scico/numpy/__init__.py

```diff
@@ -155,7 +155,10 @@ def vdot(a, b):
         _not_implemented_functions.append((name, func))
 
 _attach_wrapped_func(
-    _not_implemented_functions, _not_implemented, module_name=sys.modules[__name__]
+    _not_implemented_functions,
+    _not_implemented,
+    module_name=sys.modules[__name__],
+    fix_mod_name=False,
```

Comment:
Why the change here?

Reply:
I was messing around with how `__module__` was set for `scico.numpy` functions to get nicer docs. But in the end I didn't change it. The change here is just that I was explicit with the `fix_mod_name=False`, which is already the default.